app/src/components/pages/errorPage/errorPage.tsx

```diff
@@ -0,0 +1,58 @@
+import { headerBreakpoints } from "@/src/utils/breakpoints";
+import {
+  Box,
+  Heading,
+  Button,
+  Text,
+  Link,
+} from "@nypl/design-system-react-components";
+import Image from "next/image";
+import { useFeedbackContext } from "../../pageLayout/pageLayout";
+import useBreakpoints from "@/src/hooks/useBreakpoints";
+
+export default function ErrorPage() {
+  const { onOpen } = useFeedbackContext();
+  const { isLargerThanLargeMobile } = useBreakpoints();
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "center",
+        alignItems: "center",
```

Comment:
There are `Center` and `Flex` components that are exported from Reservoir. Can you try those so you don't have to add these style props? I'm curious to see if those help make this easier but if you prefer this, it's completely ok.

Reply:
I think you had this before but I just noticed it, sorry!